.github/workflows/add-docs-preview-link.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           statusName: "buildkite/docs-build-pr"
           # https://elasticsearch-ci.elastic.co/job/elastic+logstash+pull-request+build-docs
           # usually finishes in ~ 20 minutes
-          timeoutSeconds: 900
+          timeoutSeconds: 1800
```

Comment:
The comment and original value dont really make sense? If it typically finishes in ~20 minutes, why would the timeout be 15 minutes? Also, the URI in the comment does not seem to be correct either. I'm assuming this is somehow tracking the buildkite job? The github action history make it look like this just always goes for about 15 minutes (the timeout threshold) but it looks like buildkite jobs can finish much fater than that https://buildkite.com/elastic/docs-build-pr/builds/194330 for example?

Reply:
I think the root of my misunderstanding here is what this is meant to track and how to validate that status https://github.com/elastic/logstash/blob/eb7e1253e003a09e3297321ec760ba73193d17b8/.github/workflows/add-docs-preview-link.yml#L22

Reply:
Let me clear up the comments - there are a bunch of mixed messages here from previous commits. 

The builds take ~ 20 minutes, so the 30 minute timeout should be sufficient.

It took me a little digging to find the [right incantation](https://github.com/elastic/docs/blob/ca7de98a85cf43a544a5e3551807d2c928028902/.github/actions/docs-preview/action.yml#L32C35-L32C118) to find product specific builds - the latest one for example is https://buildkite.com/elastic/docs-build-pr/builds?created_from_days=7&meta_data%5Brepo_pr%5D=logstash_16701